config/data/courts.csv

```diff
@@ -0,0 +1,251 @@
+C,Carlisle Crown Court
+B,Carlisle Magistrates' Court
+C,Central Criminal Court
+C,Chelmsford Crown Court
+B,Chelmsford Magistrates' Court
+B,Cheltenham Magistrates' Court
+C,Chester Crown Court
+B,Chester Magistrates' Court
+B,Chesterfield Magistrates' Court
+C,Chichester Crown Court
+B,Chorley Magistrates' Court
+B,Cirencester Magistrates' Court
+B,City of London Magistrates' Court
+B,Colchester Magistrates' Court
+C,Court of Appeal
+C,Coventry Crown Court
+B,Coventry Magistrates' Court
+B,Crawley Magistrates' Court
+B,Crewe Magistrates' Court
+C,Croydon Crown Court
+B,Croydon Magistrates' Court
+B,Cwmbran Magistrates' Court
+B,Darlington Magistrates' Court
+C,Derby Crown Court
+B,Derby Justice Centre (aka Derby St Mary Adult)
+C,Doncaster Crown Court
+B,Doncaster Magistrates' Court
+B,Dudley Magistrates' Court
+C,Durham Crown Court
+B,Ealing Magistrates' Court
+B,Eastbourne Magistrates' Court
+C,Exeter Crown Court
+B,Exeter Magistrates' Court
+B,Folkestone Magistrates' Court
+B,Gateshead Magistrates' Court
+C,Gloucester Crown Court
+C,Great Grimsby Crown Court
+B,Great Yarmouth Magistrates' Court
+B,Grimsby Magistrates' Court
+C,Guildford Crown Court
+B,Guildford Magistrates' Court
+B,Harrogate Magistrates' Court
+C,Harrow Crown Court
+B,Hastings Magistrates' Court
+B,Hatfield Magistrates' Court
+B,Haverfordwest Magistrates' Court
+B,Hendon Magistrates' Court
+C,Hereford Crown Court
+B,Herefordshire Magistrates' Court
+B,Highbury Corner Magistrates' Court
+B,Horsham Magistrates' Court
+B,Huddersfield Magistrates' Court
+B,Hull Magistrates' Court
+B,Huntingdon Magistrates' Court
+C,Inner London Crown Court
+C,Ipswich Crown Court
+B,Ipswich Magistrates' Court
+B,Isle of Wight Magistrates' Court
+C,Isleworth Crown Court
+B,Kidderminster Magistrates' Court
+C,King's Lynn Crown Court
+C,Kingston upon Thames Crown Court
+C,Kingston-upon-Hull Crown Court
+C,Lancaster Crown Court
+B,Lancaster Magistrates' Court
+B,Lavender Hill Magistrates' Court
+B,Leamington Spa Magistrates' Court
+C,Leeds Crown Court
+B,Leeds Magistrates' Court
+C,Leicester Crown Court
+B,Leicester Magistrates' Court
+C,Lewes Crown Court
+C,Lincoln Crown Court
+B,Lincoln Magistrates' Court
+C,Liverpool Crown Court
+B,Liverpool Magistrates' Court
+B,Llandrindod Wells Magistrates' Court
+B,Llandudno Magistrates' Court
+B,Llanelli Magistrates' Court
+B,Loughborough Magistrates' Court
+C,Luton Crown Court
+B,Luton Magistrates' Court
+B,Maidenhead Courthouse
+C,Maidstone Crown Court
+B,Maidstone Magistrates' Court
+B,Manchester City Magistrates' Court
+C,Manchester Crown Court
+B,Mansfield Magistrates' Court
+B,Margate Magistrates' Court
+B,Medway Magistrates' Court
+C,Merthyr Tydfil Crown Court
+B,Merthyr Tydfil Magistrates' Court
+B,Milton Keynes Magistrates' Court
+C,Minshull Street Manchester Crown Court
+C,Mold Crown Court
+B,Mold Magistrates' Court
+B,Newcastle Magistrates' Court
+C,Newcastle upon Tyne Crown Court
+B,Newcastle upon Tyne Magistrates' Court

```

Comment:
Do you think these are duplicates, on lines 145 / 147? Googling around, seems like these refer to the same place.

Reply:
Yep, this is one of the courts @arthurashman was looking at. I just took the list of names from the gem and bring it over, without any changes.